CoreUtilities/src/au/gov/asd/tac/constellation/utilities/xml/XmlUtilities.java

```diff
@@ -414,21 +428,15 @@ public String getNodeAttr(final String tagName, final String attrName, final Nod
         for (int i = 0; i < nodes.getLength(); i++) {
             final Node node = nodes.item(i);
             if (node.getNodeName().equalsIgnoreCase(tagName)) {
-                final NodeList childNodes = node.getChildNodes();
-                for (int j = 0; j < childNodes.getLength(); j++) {
-                    final Node data = childNodes.item(j);
-                    if (data.getNodeType() == Node.ATTRIBUTE_NODE && data.getNodeName().equalsIgnoreCase(attrName)) {
-                        return data.getNodeValue();
-                    }
-                }
+                String result = getNodeAttr(attrName, node);  
+                if (result != null) {return result; }
```

Comment:
`result` can be final. Can you also fix the formatting of the if block?

Reply:
done